tests/unit/document_tests.py

```diff
@@ -23,6 +23,7 @@
 """
 
 import unittest
+from mock import mock
```

Comment:
I think your code will still work if you just use `import mock` here instead of `from mock import mock`.